config.go

```diff
@@ -1,9 +1,35 @@
 package ynabber
 
 import (
+	"encoding/json"
 	"time"
 )
 
+const DateFormat = "2006-01-02"
+
+type Date time.Time
+
+// Decode implements `envconfig.Decoder` for Date to parse string to time.Time
+func (date *Date) Decode(value string) error {
+	time, err := time.Parse(DateFormat, value)
+	if err != nil {
+		return err
```

Comment:
Does the user actually see a sensible error message?
I.e. unable to convert this and this to date?